docker-compose.yml

```diff
@@ -5,17 +5,34 @@ services:
     restart: always
     volumes:
       - lodestar:/root/.local/share/lodestar
-    command: beacon --network ${NETWORK:-pyrmont}
+      - /root/.local/share/lodestar/pyrmont:/root/.local/share/lodestar/pyrmont
+    command: beacon --network ${NETWORK:-pyrmont} --api.rest.host 0.0.0.0
+    expose:
+      - "9596"
```

Comment:
What do you want to achieve here? 9565 should be available to the validator service without this

> `expose`: Expose ports without publishing them to the host machine - they’ll only be accessible to linked services. Only the internal port can be specified.

https://docs.docker.com/compose/compose-file/compose-file-v3/#expose